src/NuGet.Core/NuGet.Protocol.Core.v3/HttpSource/HttpRetryHandler.cs

```diff
@@ -86,6 +86,8 @@ public HttpRetryHandler()
                             Strings.Http_RequestLog,
                             request.Method,
                             requestUri));
+                        
+                        response?.Dispose();
```

Comment:
`Dispose` in catch block as well?
https://github.com/NuGet/NuGet.Client/pull/467/files#diff-c2ed78af63778dfdcfdbb72fd86300dbR116